libs/sql-schema-describer/tests/postgres/mod.rs

```diff
@@ -19,10 +19,7 @@ pub async fn get_postgres_describer(sql: &str, db_name: &str) -> postgres::SqlSc
         .unwrap();
 
     let drop_schema = format!("DROP SCHEMA IF EXISTS \"{}\" CASCADE;", SCHEMA);
-    client
-        .query_raw(drop_schema.as_str(), &[])
-        .await
-        .expect("dropping schema");
+    client.query_raw(&drop_schema, &[]).await.expect("dropping schema");
```

Comment:
If you provide empty parameters to `query_raw` or `execute_raw`, and do not expect any results back, please use the `raw_cmd` from Quaint. It will skip the preparing step and use the text protocol, which works much better for `DROP SCHEMA` et.al.

Reply:
Will do